### PR TITLE
Issue #Mentor : fix : prod build issue fix and user side menu fix

### PR DESCRIPTION
**Pull request by pradeepkumarcm96**

### CODE REVIEW

File: src/app/core/services/user/user.service.ts [hunk 1]
```diff
@@ -36,5 +40,9 @@ export class UserService {
     const hourDifference = duration.asHours();
     return (hourDifference < 2) ? false : true;
   }
+
+  getCurrentUser(user){
+    this.userEvent.next(user);
```
Comment (kiranharidas187) [inline src/app/core/services/user/user.service.ts:45]: Why do we need a subject here?

Comment (kiranharidas187) [reply src/app/core/services/user/user.service.ts:45]: Only on login and logout we need to trigger the subject

File: src/app/shared/components/input-chip/input-chip.component.html [hunk 2]
```diff
@@ -21,8 +21,7 @@
       <ion-label>{{ chip.label }}</ion-label>
     </ion-chip>
     <ion-chip *ngIf="showAddOption" (click)="addNewOption()">
-      <ion-icon name="add" *ngIf="!selectedChips.has(chip)"></ion-icon>
-      <ion-icon name="close-circle" color="white" *ngIf="selectedChips.has(chip)"></ion-icon>
+      <ion-icon name="add"></ion-icon>
```
Comment (kiranharidas187) [inline src/app/shared/components/input-chip/input-chip.component.html:24]: is this change required?

Comment (pradeepkumarcm96) [reply src/app/shared/components/input-chip/input-chip.component.html:24]: yes on click add only need to show alert selectedchip for add is not required

File: src/app/core/services/user/user.service.ts [hunk 3]
```diff
@@ -23,6 +26,7 @@ export class UserService {
       .getLocalData(localKeys.USER_DETAILS)
       .then((data: any) => {
         this.userDetail=data;
+        this.userEvent.next(data.user);
```
Comment (kiranharidas187) [inline src/app/core/services/user/user.service.ts:29]: Why do we need this here?

File: src/app/shared/components/input-chip/input-chip.component.html (hunk 2 above)
Comment (kiranharidas187) [reply src/app/shared/components/input-chip/input-chip.component.html:24]: I think we need add and close icons. @rejneesh1  Please confirm

File: src/app/core/services/user/user.service.ts (hunk 3 above)
Comment (pradeepkumarcm96) [reply src/app/core/services/user/user.service.ts:29]: after login close app and open we need to to set the user data to the subject to show in side menu

File: src/app/shared/components/input-chip/input-chip.component.html (hunk 2 above)
Comment (rejneesh1) [reply src/app/shared/components/input-chip/input-chip.component.html:24]: @kiranharidas187  this should be fine , its removed for add chip (which always need to  show add icon only)
